packages/docs/src/components/landing/Header.vue

```diff
@@ -160,6 +160,10 @@ export default class Header extends Vue {
     justify-content: space-between;
   }
 
+  ::v-deep(.va-dropdown__content) {
+    background: #f6f8f9 !important;
```

Comment:
Is that truly needed or can we expand buttons api somehow / maybe use css variables?

Reply:
for next design iteration